src/main/java/org/opentripplanner/updater/trip/TimetableSnapshotSource.java

```diff
@@ -1123,19 +1137,17 @@ private Result<UpdateSuccess, UpdateError> handleModifiedTrip(
   private Result<UpdateSuccess, UpdateError> handleCanceledTrip(
     FeedScopedId tripId,
     final LocalDate serviceDate,
-    CancelationType markAsDeleted
+    CancelationType markAsDeleted,
```

Comment:
The variable name seems confusing - was this converted from a boolean at some point? Maybe it should just be called `cancelationType`.

Reply:
I can rename this.